include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -10,20 +10,44 @@
 #include <limits>
 #include <stdexcept>
 
-
+/**
+ * Namespace of converters designed for better performance
+ */
 namespace SuperiorMySqlpp { namespace Converters
 {
     namespace detail
     {
+        /**
+         * @brief Calculates the power of number
+         * @tparam T type of output and base
+         * @param x base
+         * @param n nonnegative exponent
+         * @return power of x
+         */
         template <class T>
         inline constexpr T pow(T const& x, std::size_t n) {
             return n > 0 ? x * pow(x, n - 1) : 1;
         }
-
+        /**
+         * @class ToIntegerParserImpl
+         * @file to_integer.hpp
+         * @brief Parses the string into the number of given type
+         * @tparam T type of result
+         * @tparam length length of the string
+         * @tparam validate whether to check if string is convertable
+         */
         template<typename T, int length, bool validate>
         struct ToIntegerParserImpl
         {
+            /**
+             * Check if length is not greater than is allowed for given type
+             */
             static_assert(std::numeric_limits<T>::digits10, "length > digits10");
+            /**
+             * @brief Parses string into number of given type
+             * @param result result to change recursively
```

Comment:
Please write first letter of sentence in upper case. Also the final dot on sentence's end is beneficial. It much more readable in generated documentation by Doxygen.